modules/minimum-required-permissions-ipi-gcp-xpn.adoc

```diff
@@ -39,7 +39,7 @@ Ensure that the host project applies one of the following configurations to the
 * `roles/compute.networkUser`
 ====
 
-If you do not supply a service account for control plane nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project. If you do not supply a service account for compute nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project for cluster destruction.
+If you do not supply a service account for control plane nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project. If you do not supply a service account for compute nodes in the `install-config.yaml` file, please grant the below permissions to the service account in the host project for cluster destruction. If you do supply service accounts for control plane and compute nodes, you do not need to grant the below permissions.
```

Comment:
Can I ask you to fix this one? Per ISG: [Do not use to indicate a relative location in a document, as in “the information below”](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#b)
```suggestion
If you do not supply a service account for control plane nodes in the `install-config.yaml` file, please grant the following permissions to the service account in the host project. If you do not supply a service account for compute nodes in the `install-config.yaml` file, please grant the following permissions to the service account in the host project for cluster destruction. If you do supply service accounts for control plane and compute nodes, you do not need to grant the following permissions.
```

Reply:
Updated, thanks